tests/pytorch_tests/function_tests/test_torch_utils.py

```diff
@@ -0,0 +1,89 @@
+#  Copyright 2024 Sony Semiconductor Israel, Inc. All rights reserved.
+#  #
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#  #
+#      http://www.apache.org/licenses/LICENSE-2.0
+#  #
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+#  ==============================================================================
+
+import unittest
+import numpy as np
+import torch
+from unittest.mock import patch
+
+from model_compression_toolkit.core.pytorch.utils import to_torch_tensor, torch_tensor_to_numpy
+
+
+class TestTorchUtils(unittest.TestCase):
+    def setUp(self):
+        # Setup any common resources
+        self.numpy_array = np.array([1.0, 2.0, 3.0])
+        self.torch_tensor = torch.tensor([1.0, 2.0, 3.0])
+        self.list_of_numbers = [1, 2, 3]
+        self.tuple_of_numbers = (1, 2, 3)
+
+    @patch('model_compression_toolkit.core.pytorch.pytorch_device_config.get_working_device')
```

Comment:
What is the purpose of this notation?

Reply:
The @patch is used to temporarily replace the behavior of a function or object during the execution of a unit test. In this case, when the test encounters a call get_working_device, it won't execute the actual function, but the mock object.